src/app/views/query-response/snippets/snippets-helper.tsx

```diff
@@ -7,23 +7,37 @@ import { getSnippet } from '../../../services/actions/snippet-action-creator';
 import { Monaco } from '../../common';
 import { genericCopy } from '../../common/copy';
 
+import { telemetry } from '../../../../telemetry';
+import { BUTTON_CLICK_EVENT, TAB_CLICK_EVENT } from '../../../../telemetry/event-types';
+import { IQuery } from '../../../../types/query-runner';
+
 interface ISnippetProps {
   language: string;
+  sampleQuery: IQuery;
```

Comment:
The sample query is already present as a redux property in the snippets component. We do not need to pass it in as a property